n2y/notion_mocks.py

```diff
@@ -144,3 +144,47 @@ def mock_select_option(name, **kwargs):
 
 def mock_relation_value():
     return {"id": mock_id()}
+
+
+def mock_page(title="Mock Title"):
+
+    # Returns a mock version the data that would be returned by
+    # the Notion API on a client.get_page() request.
+
+    user = mock_user()
+    created_time = datetime.now().isoformat()
+    notion_id = mock_id()
+    hyphenated_title = title.replace(" ", "-")
+
+    page_object = {
```

Comment:
Small styling thing, but to follow other functions in this file:

```suggestion
    return {
```

Reply:
Done